### PR TITLE
Add Vue template parser/transformer

### PR DESCRIPTION
**Pull request by kindy**
Add Vue template parser.
can preview here: https://kindy.github.io/astexplorer/ .
select "Vue" type.

also, there is a transformer to convert Vue template to JS render functions.

### CODE REVIEW

**Comment by kindy** [comment]
due to #286 issue, I use prettier v1.5.2 in my build.

**Comment by kindy** [comment]
@fkling  ping .

File: website/src/parsers/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const localRequire = require.context('./', true, /^\.\/(?!utils)[^/]+\/(transformers\/([^/]+)\/)?(codeExample\.txt|[^/]+?\.js)$/);
 
+const a = 1;
```
Comment (fkling) [inline website/src/parsers/index.js]: Please remove.

File: website/src/parsers/vue/transformers/vue-template-compiler/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      'recast',
+    ], (compiler, prettier, babel, recast) => callback({compiler, prettier, babel, recast}));
+  },
+
+  transform({compiler, prettier, babel, recast}, transformCode, code) {
+    let {ssr, ...options} = compileModule( // eslint-disable-line no-shadow
+      transformCode
+    );
+
+    function methodNameFix(babel) {
```
Comment (fkling) [inline website/src/parsers/vue/transformers/vue-template-compiler/index.js]: This could be defined outside the `transform` method.

**Comment by kindy** [comment]
@fkling  fixed.
can you have look ?
Thanks.

**Comment by fkling** [comment]
This is too old to merge now. Vue was added as a parser at least.

**Pull request closed without merging**